### PR TITLE
Update etcd for v0.4.0-beta.2

### PR DESCRIPTION
**Pull request by cvlc**

### CODE REVIEW

File: charts/etcd-cluster-operator/templates/etcdcluster_cr.yaml [hunk 1]
```diff
@@ -48,16 +48,24 @@ spec:
...
-              topologyKey: kubernetes.io/hostname
+        - weight: 1
+          podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+              - key: etcd.improbable.io/cluster-name
+                operator: In
+                values:
+                  - storageos-etcd
+            topologyKey: topology.kubernetes.io/zone
```
Comment (Mojachieee) [inline charts/etcd-cluster-operator/templates/etcdcluster_cr.yaml:68]: You might want to parameterise this to allow users to set their own "topology key" value

**Review by Mojachieee — DISMISSED** [review]
LGTM - left one comment

**Review by aeroniero33 — DISMISSED** [review]
LGTM!

**Review by aeroniero33 — APPROVED** [review]
LGTM!

**Pull request merged**